Header header
int agentID
float64[] obstacles_x
float64[] obstacles_y
float64 x
float64 y
float64 z
float64 theta